data_steward/cdr_cleaner/cleaning_rules/drop_participants_without_completed_basics.py

```diff
@@ -54,26 +57,23 @@
     ON (concept_id = ancestor_concept_id)
   JOIN `{{project}}.{{dataset}}.observation`
     ON (descendant_concept_id = observation_concept_id)
-  JOIN `{{project}}.{{dataset}}._mapping_observation`
-    USING (observation_id)
   WHERE concept_class_id = 'Module'
     AND concept_name IN ('The Basics')
-    AND src_hpo_id = 'rdr'
     AND questionnaire_response_id IS NOT NULL)
 """)
 
 
-class DropParticipantsWithoutPPI(DropMissingParticipants):
+class DropParticipantsWithoutCompletedBasics(DropMissingParticipants):
```

Comment:
The name is much better.  But I might suggest changing it to `DropParticipantsWithoutAnyBasics` because as long as the participant has answered at least one question from The Basics and we have that data, we will not drop that participant.  They don't technically have to complete the entire survey.